sktime/utils/_testing/annotation.py

```diff
@@ -10,7 +16,17 @@ def make_annotation_problem(
     make_X=False,
     n_columns=2,
     random_state=None,
+    estimator_type=None,
 ):
+    if estimator_type == "Poisson":
+        rng = check_random_state(random_state)
```

Comment:
No you should leave `random_state`! My only point was that since you pass it into `piecewise_poisson` anyway (and run the check there) that there is no need to also run the check here too! :)

Reply:
Ah I see, nevermind!  I missed that you were using this to generate random `lambdas` and `lengths`!  Sorry, oversight on my part. Yes this is a fine way to do it.